app/controllers/application_controller.rb

```diff
@@ -20,6 +20,7 @@ def conversation
   protected
 
   def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:name])
     devise_parameter_sanitizer.permit(:account_update, keys: [:name])
```

Comment:
Why are you adding sanitizers for signup?

Reply:
Added it to be able to signup to the app on localhost:3000 and troubleshoot